async-clipboard/demo.js

```diff
@@ -0,0 +1,55 @@
+const copy = document.querySelector('#copy');
+const paste = document.querySelector('#paste');
+const out = document.querySelector('#out');
+
+/** Write contents of the textarea to the clipboard when clicking "Copy" */
+copy.onclick = () => {
+  navigator.clipboard.writeText(out.value)
+    .then(result => {
+      ChromeSamples.log('Text copied.' + result);
+    })
+    .catch(err => {
+      ChromeSamples.log('Failed to copy text: ' + err);
+    });
+};
+
+/** Read from clipboard when clicking the Paste button */
+paste.onclick = () => {
+  navigator.clipboard.readText()
+    .then(text => {
+      out.value = text;
+      ChromeSamples.log('Text pasted.');
+    })
+    .catch(err => {
+      ChromeSamples.log('Failed to read from clipboard: ' + err);
+    });
+};
+
+/** Watch for pastes */
+document.addEventListener('paste', e => {
+  e.preventDefault();
+  navigator.clipboard.getText().then(text => {
+    ChromeSamples.log('Updated clipboard contents: ' + text);
+  });
+});
+
+/** Set up buttons to request permissions and display status: */
+document.querySelectorAll('data-permission').forEach(btn => {
```

Comment:
This code currently doesn't work. It should be `document.querySelectorAll('[data-permission]')`

Reply:
Indeed! I've managed to get Jekyll set up now and things are working properly.